radio/src/gui/480x272/widgets/outputs.cpp

```diff
@@ -36,18 +36,14 @@ class OutputsWidget: public Widget
 
     uint8_t drawChannels(uint16_t x, uint16_t y, uint16_t w, uint16_t h, uint8_t firstChan)
     {
-      char chanString[] = "CH32";
+      char chanString[] = "CH32 123456";
```

Comment:
Done in commit https://github.com/opentx/opentx/pull/4147/commits/e5c8cbe0cc422fee1709c93add8990e6052248f7